src/components/PlayerProfile.vue

```diff
@@ -162,17 +165,19 @@ export default {
     $route: 'executeQuery',
   },
   methods: {
+    fromNow(date) {
+      return `${countdown(new Date(), date, countdown.ALL, 1)} ago`;
+    },
     formatDate(date) {
-      if (date == null) {
+      if (!date) {
         return '-';
       }
       const d = new Date(date);
-      return (d.getFullYear() === 1969) ? 'Never' : d.toLocaleString();
+      return d.toLocaleString();
     },
     getRanks() {
```

Comment:
Could this be a computed? Then we also don't have to call it in the template.

Reply:
Due to the way the function is used, I don't think it can be computed.